tests/zfs-tests/tests/functional/cli_root/zfs_mount/zfs_mount_remount.ksh

```diff
@@ -44,7 +44,7 @@ verify_runnable "both"
 
 function cleanup
 {
-	log_must zpool export $TESTPOOL
+	log_must_busy zpool export $TESTPOOL
```

Comment:
Did you mean to include this in the PR?  It looks like a fix for the occasional `zfs_mount_remount` failures we see under Fedora 26 (which I'm all for fixing).

Reply:
Yes: i've seen it failing quite a lot on f26 and that seems to fix it. I tried to sneak it in even if it has nothing to do with the main issue.

Since now we have other recent issues reported on the ZTS (https://github.com/zfsonlinux/zfs/issues/6627) i'm ok with dropping this here and push it in a "ZTS-only fixes" PR.

Reply:
Got it.  Yeah, let's drop it from this PR and make the fix in the context of #6627.